OtripleS.Web.Api/Models/Students/Exceptions/StudentServiceException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace SchoolEM.Models.Students.Exceptions
+{
+    public class StudentServiceException : Exception
+    {
+        public StudentServiceException(Exception innerException)
+            : base("System error occurred, contact support.", innerException)
```

Comment:
Better to get this all string from a string Resource file, so that going forward we can hook up the internationalization with the project to support major languages.